doc/doxygen/header.tex

```diff
@@ -128,7 +128,7 @@
     {\Large Unified Communication X (UCX)}\\
 \vspace*{0.5cm}
 {\large API Standard}\\
-{\small Version 1.3}\\
+{\small Version 1.3.1}\\
```

Comment:
API does not contain minor version, since API does not change between minor versions